plugin-server/src/worker/vm/extensions/posthog.ts

```diff
@@ -41,6 +42,7 @@ async function queueEvent(hub: Hub, pluginConfig: PluginConfig, data: InternalDa
                     now: data.timestamp,
                     sent_at: data.timestamp,
                     uuid: data.uuid,
+                    ...(data.groups ? { groups: data.groups } : {}),
```

Comment:
This doesn't work - we don't send groups information under top-level of the message. it's supposed to be data.properties.$groups